src/main/resources/application.yaml

```diff
@@ -12,6 +12,10 @@ spring:
 
 bulk-scan-processor-url: ${BULK_SCAN_PROCESSOR_URL:}
 
+queue:
+  conn-string: ${QUEUE_CONNECTION_STRING:}
```

Comment:
how about: connection-string
you're doing the longer version in the env variable anyway, may as well be consistent

Reply:
done